src/invidious/channels/community.cr

```diff
@@ -155,14 +157,22 @@ def fetch_channel_community(ucid, continuation, locale, format, thin_mode)
                         json.field "authorUrl", author_info["navigationEndpoint"]["commandMetadata"]["webCommandMetadata"]["url"]
 
                         # TODO: json.field "authorThumbnails", "channelThumbnailSupportedRenderers"
-                        # TODO: json.field "authorVerified", "ownerBadges"
+                        json.field "authorVerified", has_verified_badge?(attachment["ownerBadges"]?).to_s
 
-                        published = decode_date(attachment["publishedTimeText"]["simpleText"].as_s)
+                        if published_time_text = attachment.dig?("publishedTimeText", "simpleText").try &.as_s
+                          published = decode_date(published_time_text)
+                          json.field "published", published.to_unix
+                          json.field "publishedText", translate(locale, "`x` ago", recode_date(published, locale))
+                        end
```

Comment:
For compatibility, can you please keep the fields, but put dummy values in here (0 for timestamp, and empty string for text). I'd prefer if we don't hard break other downstream projects!

Reply:
Sounds good, i made the change for the lengthSeconds as well 🙂